frontend/src/components/layout.ts

```
@@ -228,92 +209,105 @@ type NModalProps = {
       switch (id) {
-        case "positive":
-          props.on.positive_click();
-          break;
-        case "negative":
-          props.on.negative_click();
-          break;
-        default:
-          // TODO: never called, see not on on.close prop
-          props.on.close();
-          break;
+        case "positive": on.positive_click();
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)